source/dhtslib/sam.d

```diff
@@ -728,6 +728,118 @@ private char* reverse(char* str)
     return str;
 }
 
+struct SamWriter{
```

Comment:
Please match codestyle of existing code.
capitalize SAM in class and struct names
put opening bra ce on next line

Reply:
Done eb250e8 .